Library/Homebrew/cask/dsl.rb

```diff
@@ -540,6 +547,30 @@ def livecheckable?
       @livecheck_defined == true
     end
 
+    def no_autobump!(because:)
+      if !because.is_a?(String) && (!because.is_a?(Symbol) || !NO_AUTOBUMP_REASONS_LIST.key?(because))
```

Comment:
Please add a `sig` to enforce String/Symbol and then just use the `NO_AUTOBUMP_REASONS_LIST` check if it's a Symbol